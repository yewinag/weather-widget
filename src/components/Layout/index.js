import styled from 'styled-components';

const Layout = styled.div`    
    display: ${props => props.flex ? "flex" : "block"};;      
    margin-bottom: ${props => props.margin ? "20px" : "0"};
    justify-content: ${props => props.justify ? "space-between" : ""};
    min-height: ${props => props.section && "400px"};
    border: ${props => props.section && "1px solid #ddd"};
    padding: ${props => props.section && "20px"};
`;

export default Layout;