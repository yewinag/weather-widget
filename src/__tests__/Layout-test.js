import React from 'react';

import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';

import Layout from '../components/Layout';

Enzyme.configure({ adapter: new Adapter() });

describe('layout', () => {
    it('render layout', () => {
        const wrapper = shallow(<Layout />)
        expect(wrapper.exists()).toBe(true);
        expect(wrapper).toMatchSnapshot();
    });
});