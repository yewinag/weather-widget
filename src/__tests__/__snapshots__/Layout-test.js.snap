// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layout render layout 1`] = `ShallowWrapper {}`;
